package.json

```diff
@@ -29,5 +30,14 @@
     "which": "^1.1.1"
   },
   "homepage": "https://github.com/jspm/npm",
-  "devDependencies": {}
+  "devDependencies": {
+    "istanbul": "0.3.21",
+    "lodash": "3.10.1",
+    "magicpen-media": "1.5.1",
+    "messy": "6.6.2",
+    "mocha": "2.3.3",
+    "unexpected": "9.16.0",
+    "unexpected-messy": "4.8.1",
```

Comment:
You should be able to cut `unexpected-messy`, `messy`, and `magicpen-media` from the explicit `devDependencies` list, even though `npm install --save-dev` adds them when you're using npm 2.